docs/advanced-node-scheduling.asciidoc

```diff
@@ -0,0 +1,342 @@
+[id="{p}-advanced-node-scheduling"]
+
+== Advanced Elasticsearch node scheduling
+
+Elastic Cloud on Kubernetes (ECK) offers full control over cluster nodes scheduling by combining Elasticsearch configuration with Kubernetes scheduling options:
+
+* defining Elasticsearch nodes roles
+* pod affinity and anti-affinity
+* availability zone and rack awareness
+* hot-warm-cold architecture
+
+These features can be combined together, to deploy a production-grade Elasticsearch cluster.
+
+=== Defining Elasticsearch nodes roles
+
+Elasticsearch nodes can be configured with link:https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html[one or multiple roles]. This allows you to describe an Elasticsearch cluster with, for example, 3 dedicated master nodes:
```

Comment:
Prefer active voice wherever possible:

`You can configure Elasticsearch nodes with link:https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html[one or multiple roles]. This allows you to describe an Elasticsearch cluster with 3 dedicated master nodes, for example:`